### PR TITLE
Update import schema logic implementation and optimize -help display information

### PR DESCRIPTION
**Pull request by 2b3c511**

### CODE REVIEW

**Comment by 2b3c511** [comment]
image

File: iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ExportSchema.java [hunk 1]
```diff
@@ -201,7 +201,6 @@ private static Options createOptions() {
 
     Option opTargetFile =
         Option.builder(TARGET_DIR_ARGS)
-            .required()
```
Comment (OneSizeFitsQuorum) [inline iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ExportSchema.java:204]: seems line 207 still contains "required"

File: iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ImportSchema.java [hunk 2]
```diff
@@ -103,7 +103,6 @@ private static Options createOptions() {
 
     Option opFile =
         Option.builder(FILE_ARGS)
-            .required()
```
Comment (OneSizeFitsQuorum) [inline iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ImportSchema.java:106]: seems line 110 still contains required

File: iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ImportSchema.java [hunk 3]
```diff
@@ -124,12 +122,12 @@ private static Options createOptions() {
             .build();
     options.addOption(opFailedFile);
 
-    Option opAligned =
-        Option.builder(ALIGNED_ARGS)
-            .longOpt(ALIGNED_ARGS)
-            .desc("Whether import schema as aligned timeseries(optional)")
-            .build();
-    options.addOption(opAligned);
+    //    Option opAligned =
```
Comment (OneSizeFitsQuorum) [inline iotdb-client/cli/src/main/java/org/apache/iotdb/tool/ImportSchema.java:125]: just remove? do not comment code

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**